botrest/src/main/resources/application.yml

```diff
@@ -1,5 +1,5 @@
 spring:
   datasource:
-    url: jdbc:mysql://localhost:3306/BlackBookRobotDB
-    username: root
-    password: 123ZOOMzoom890
```

Comment:
It is temporary solution. Later h2 will be used only for integration tests (and tests will be provided with db access properties) and here we will put data to mysql db.
Eventually there will be another application which will be holding all configuration data for all modules.